crates/swc_ecma_ast/src/expr.rs

```diff
@@ -157,8 +157,8 @@ pub enum Expr {
     #[tag("TsInstantiation")]
     TsInstantiation(TsInstantiation),
 
-    #[tag("TsSatisfactionExpr")]
-    TsSatisfaction(TsSatisfactionExpr),
+    #[tag("TsSatisfiesExpression")]
+    TsSatisfies(TsSatisfiesExpr),
```

Comment:
I will no longer have Benny Benassi stuck in my head while working with this. Thank you!